.dockerignore

```diff
@@ -1,12 +1,11 @@
-################################################################################
-# Repo
+# Ignore everything by default
+*
 
-.circleci/
-.devcontainer/
-.dockerignore
-.git/
-.github/
-.gitignore
-**.Dockerfile
-**Dockerfile
-doc/
+# First-order allow exception for select directories
```

Comment:
Isn't this essentially everything?

Reply:
White listing is easier than black listing at this point. Second order exceptions make this simpler to track.
E.g. prevents random stuff in the repo from breaking the build cache unintentionally while working.